README.md

```diff
@@ -471,6 +509,20 @@ CMS are a practical way to setup a website with many features. CMS often come wi
 
 See https://staticsitegenerators.net and https://www.staticgen.com
 
+
+## Ticketing systems
+*Web-based ticketing system.*
```

Comment:
I think we should differentiate between ticketing (as in sysadmin) and bug tracking (as in developer).

How about splitting the Bug and Task trackers category into bug trackers and "task trackers and ticketing systems", moving the latter to the top level and linking to it from software dev?